packages/gestalt/src/SearchGuide.tsx

```diff
@@ -0,0 +1,191 @@
+   */
+  thumbnail?:
+    | { avatar: ReactElement }
+    | { avatarGroup: ReactElement }
+    | { image: ReactElement }
+    | { icon: ReactElement };
+};
+
+/**
+ * [SearchGuide](https://gestalt.pinterest.systems/web/searchguide) appends and refines a search query. They appear under [SearchField](/web/searchfield) after user submits a search input.
+ *
+ * ![SearchGuide light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/SearchGuide.spec.ts-snapshots/SearchGuide-chromium-darwin.png)
+ * ![SearchGuide dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/SearchGuide-dark.spec.ts-snapshots/SearchGuide-dark-chromium-darwin.png)
+ *
+ */
+
+const SearchGuideWithForwardRef = forwardRef<HTMLButtonElement, Props>(function SearchGuide(
+  {
+    accessibilityControls,
+    accessibilityExpanded,
+    accessibilityHaspopup,
+    accessibilityLabel,
+    color = '01',
+    dataTestId,
+    expandable,
+    onClick,
+    selected = false,
+    text,
+    thumbnail,
+  }: Props,
+  ref,
+) {
+  const isInVRExperiment = useInExperiment({
+    webExperimentName: 'web_gestalt_visualRefresh',
+    mwebExperimentName: 'web_gestalt_visualRefresh',
+  });
+
+  const innerRef = useRef<null | HTMLButtonElement>(null);
+
+  // When using both forwardRef and innerRef, React.useimperativehandle() allows a parent component
+  // that renders <SearchGuide ref={inputRef} /> to call inputRef.current.focus()
+  // @ts-expect-error - TS2322 - Type 'HTMLButtonElement | null' is not assignable to type 'HTMLButtonElement'.
+  useImperativeHandle(ref, () => innerRef.current);
+  const { isFocusVisible } = useFocusVisible();
+
+  const buttonClasses = isInVRExperiment
+    ? classnames(styles.searchguideVr, touchableStyles.tapTransition, {
+        [styles[`color${color}`]]: !selected,
+        [focusStyles.hideOutline]: !isFocusVisible,
+        [styles.vrFocused]: isFocusVisible,
+        [styles.selectedVr]: selected,
+      })
+    : classnames(styles.searchguide, touchableStyles.tapTransition, [styles[`color${color}`]], {
+        [styles.selected]: selected,
+        [focusStyles.hideOutline]: !isFocusVisible && !selected,
+        [focusStyles.accessibilityOutline]: isFocusVisible,
+      });
+  const childrenDivClasses = classnames(styles.childrenDiv);
+
+  const textComponent = (
+    <TextUI
+      align="center"
+      color={isInVRExperiment && selected ? 'inverse' : 'dark'}
+      overflow="noWrap"
+    >
+      {text}
+    </TextUI>
+  );
+  const thumbnailVariant = thumbnail && (
+    <Flex alignItems="center" gap={{ row: 2, column: 0 }} justifyContent="center">
+      {'avatar' in thumbnail && (
+        <Box aria-hidden marginStart={isInVRExperiment ? 2 : undefined} minWidth={40}>
+          {cloneElement(thumbnail.avatar, { size: 'fit' })}
```

Comment:
could we reuse 

```
import {
  AvatarThumbnail,
  IconThumbnail,
  ImageThumbnail,
  Message,
  TypeThumbnail,
} from './sharedSubcomponents/thumbnailSubcomponents';

```

Reply:
It would require some changes in thumbnailSubcomponents, there's some hardcoded props on it that would break SearchGuide design. 
This might be a good follow up PR.

Reply:
I'll also leave the avatar border for this further look into Toast.